src/redux/books/books.jsx

```diff
@@ -0,0 +1,75 @@
+import axios from 'axios';
+
+const ADD_BOOK = 'ADD_BOOK';
+const REMOVE_BOOK = 'REMOVE_BOOK';
+const GET_BOOK = 'GET_BOOK';
+const API_URL = 'https://us-central1-bookstore-api-e63c8.cloudfunctions.net/bookstoreApi/apps/OoMg7JW7xItdmWuHqq1t/books/';
+
+const addBook = (id, title, author) => (dispatch) => {
+  axios.post(API_URL, {
+    item_id: id,
+    title,
+    author,
+    category: 'Fiction',
+  }).then(() => {
+    dispatch({
+      type: ADD_BOOK,
+      book: {
+        id,
+        title,
+        author,
+      },
+    });
+  });
+};
+
+const removeBook = (id) => (dispatch) => {
+  const deleteUrl = `${API_URL}${id}`;
+  axios.delete(deleteUrl, {
+    item_id: id,
+  }).then(() => {
+    dispatch({
+      type: REMOVE_BOOK,
+      id,
+    });
+  });
+};
```

Comment:
- [ ] Kindly use Redux Toolkit and the createAsyncThunk for interacting with the API as it's required in the project directory for getting and pushing book to the API👍